UnitTests/GitCommands.Tests/CommitMessageManagerTests.cs

```diff
@@ -21,7 +21,7 @@ public class CommitMessageManagerTests
         private ReferenceRepository _referenceRepository;
 
         private readonly string _workingDirGitDir = @"c:\dev\repo\.git";
-        private readonly Encoding _encoding = Encoding.UTF8;
+        private readonly Encoding _encoding = new UTF8Encoding(false);
```

Comment:
What is the motivation for this change?

Reply:
To disable BOM also for this test. It's not mandatory, but since I already handled similar cases, I preferred to have no BOM at all.

Reply:
I suggest adding a comment to spare others from searching this in docs.

Reply:
```suggestion
        private readonly Encoding _encoding = new(encoderShouldEmitUTF8Identifier: false);
```
makes it clear.